### DISCUSSION TITLE
[#991887/995503] Gamemode command

### DISCUSSION THREAD

**Discussion by SmokeyStack** [Mojang/minecraft-editor#40] [Bugs]
It makes no sense that using `/gamemode` in the chat is disabled but using via command blocks is perfectly ok. This can occur in a soft lock similar to #20 .

Repro steps
1. Open Chat
2. Type /gamemode

Observed result vs. what you actually expected to happen
The chat gives the autofill options. If it is unsupported, the autofill command should not work. Similar enough, command blocks should also not be allowed to execute said command.

**Comment by TORD31** [comment]
It's also possible to change the gamemode in the menu which also shouldn't be possible. See here: https://1drv.ms/v/s!Asr1UnK1Sz_LgQRMZrYTuN8dw7iz?e=U2erjl

**Reply by minecraftchloe** [reply]
> It's also possible to change the gamemode in the menu which also shouldn't be possible. See here: https://1drv.ms/v/s!Asr1UnK1Sz_LgQRMZrYTuN8dw7iz?e=U2erjl

@TORD31 Can't access that file :(

**Reply by TORD31** [reply]
I tried it agian. Maybe this link work: https://1drv.ms/v/s!Asr1UnK1Sz_LgQX7NARqx8K_VUlS?e=igqryG

Or maybe you don't need the video to confirm the issue

**Reply by minecraftchloe** [reply]
Works! Thank you!

**Comment by rpoerschke** [comment]
Fixed in 0.3.3.  Thanks!
